sdks/java/core/src/main/java/org/apache/beam/sdk/util/ZipFiles.java

```diff
@@ -226,6 +228,8 @@ static void zipDirectory(
    * @throws IOException the zipping failed, e.g. because the input was not
    *     readable.
    */
+  @SuppressFBWarnings(value = "NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE",
```

Comment:
The inline comment doesn't work there because the dereference is actually inside the syntactic sugar of the for-each loop.